tailwind.config.mjs

```diff
@@ -14,8 +14,10 @@ export default {
         'slide-in-from-right': 'slide-in-from-right 0.3s forwards ease-in-out',
       },
       borderRadius: {
-        // Equivalent to tailwind default `rounded-sm` size
-        DEFAULT: '0.125rem',
+        // Equivalent to tailwind default.
+        // Remove once frontend-shared preset does not define its own config
+        DEFAULT: '0.25rem',
+        lg: '0.5rem',
```

Comment:
I was planning to remove this config and add the CSS variables instead, but since we don't have any CSS file in this project, this approach was easier.